trigger-openqa_in_openqa

```diff
@@ -22,12 +22,20 @@ build_tag=${BUILD_TAG:-}
 client_prefix=${client_prefix:-}
 full_run=${FULL:-}
 group_id="${group_id:-"openQA"}"
+osc=${osc:-osc}
+src_project=${dst_project:-devel:openQA}
+staging_project=${dst_project:-devel:openQA:testing}
+dst_project=${dst_project:-devel:openQA:tested}
```

Comment:
Is there a use-case to allow overriding all the project to the same value and have `$src_project` and `$dst_project` the same or is it a typo?

Reply:
It is a typo :-)